src/lib/statefulForm.jsx

```diff
@@ -142,6 +143,16 @@ export default function statefulForm(mapPropsToFormConfig) {
           delete fieldsFromConfig.advancedFields
         }
 
+        // FIXME Handle folders like a normal field
+        if (Array.isArray(configFolders) && configFolders.length) {
+          const folder = configFolders[0] // we only handle the first one for now
+          if (folder.defaultDir) {
+            fieldsFromConfig.folderPath = {
```

Comment:
Is it still relevant to keep this logic ? What the matter ?

Reply:
I would prefer to be retrocompatible as much as possible